web/src/components/tasks/business-formation/BusinessFormation.test.tsx

```diff
@@ -310,10 +310,13 @@ describe("<BusinessFormation />", () => {
       legalStructureId,
     });
     const formationData = generateEmptyFormationData();
+    formationData.formationFormData.addressMunicipality = generateMunicipality({
+      displayName: "Newark",
+      name: "Newark",
+    });
     const page = preparePage({
       business: { profileData, formationData },
       displayContent,
-      municipalities: [generateMunicipality({ displayName: "Newark", name: "Newark" })],
```

Comment:
Should we modify `preparePage` not to take `municipalities` as a prop and remove the associated logic?

Reply:
Updated